Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM golang:1.6
-
-RUN apt-get update && \
-    apt-get install -y apache2-utils && \
-    rm -rf /var/lib/apt/lists/*
+FROM golang:1.6-alpine
```

Comment:
If go 1.6.2 is released, and we are using `golang:1.6.1-alpine` upgrading will be more painful.  (a fact I'm discovering with the go 1.5.4 release)